test/CMakeLists.txt

```diff
@@ -105,6 +106,32 @@ COMMAND IOOMEZarrNGFFTestDriver
     ${ITK_TEST_OUTPUT_DIR}/cthead1Subregion.mha
 )
 
+itk_add_test(
+  NAME IOOMEZarrNGFF_readTimeIndex0
+  COMMAND IOOMEZarrNGFFTestDriver
+    --compare
+      DATA{Baseline/simple_tyx_t0.mha}
+      ${ITK_TEST_OUTPUT_DIR}/simple_tyx_t0.mha
+    itkOMEZarrNGFFReadSliceTest
+      DATA{Input/simple_tyx.zarr.zip}
```

Comment:
You forgot to add this image (`simple_tyx.zarr.zip`) to git.

Reply:
I see it here in the PR files, is it visible for you? https://github.com/InsightSoftwareConsortium/ITKIOOMEZarrNGFF/pull/49/files#diff-a884acf9d27f8c07da55820013d4cea8ed4b0be4118141e521c1206190d7b4db

Reply:
Oops, you are right - I didn't see it there in the middle.